app/controllers/reservations_controller.rb

```diff
@@ -1,14 +1,12 @@
 # frozen_string_literal: true
```

Comment:
This controller looks like the perfect point to start refactorization to MonteWay - it's really bloated 😢

Reply:
yeah, I can tell 😒 This is what I'm going to do next